pyfolio/plotting.py

```diff
@@ -94,10 +94,10 @@ def context(context='notebook', font_scale=1.5, rc=None):
                                 rc=rc)
 
 
-def plot_rolling_risk_factors(
+def plot_rolling_fama_french(
         returns,
         risk_factors=None,
-        rolling_beta_window=63 * 2,
+        rolling_window=63 * 2,
```

Comment:
How about we refactor this to:
* Define a global variable `BDAYS_MONTH = 21`
* The default will become `BDAYS_MONTH * 6`